### PR TITLE
Brut force all possible lockfile and sbom file names.

### PR DESCRIPTION
**Pull request by mohsinenar**
- Brute force file output.
- Fix unittest.
- Fix unittest.

### CODE REVIEW

File: agent/osv_agent.py [hunk 1]
```diff
@@ -20,6 +43,62 @@
...
+    """Perform the osv scan with two flags with --sbom and --lockfile,  letting OSV validate the file
+     instead of guessing the file format.
+    Args:
+        path: The file path.
+        content: Scanned file content
+    """
+    file_name = pathlib.Path(path).name
+    decoded_content = content.decode("utf-8")
+    with open(file_name, "w", encoding="utf-8") as file_path:
+        file_path.write(decoded_content)
```
Comment (3asm) [inline agent/osv_agent.py:93]: If you are using pathlib, you can directly write bytes on that object.

File: agent/osv_agent.py [hunk 2]
```diff
@@ -20,6 +43,62 @@
...
+
+    for command in _construct_commands(file_name):
+        output = _run_command(command)
+        if output is not None:
+            return output
```
Comment (3asm) [inline agent/osv_agent.py:98]: Does it make sense to bailout of the first one to not return None?

**Review by 3asm — APPROVED** [review]

File: agent/osv_agent.py (hunk 2 above)
Comment (mohsinenar) [reply agent/osv_agent.py:98]: The output gets validated before returning None.

Comment (3asm) [reply agent/osv_agent.py:98]: ok.

**Pull request merged**